app/assets/javascripts/discourse/app/components/d-modal.hbs

```diff
@@ -21,87 +17,84 @@
     {{will-destroy this.cleanupListeners}}
     {{trap-tab preventScroll=false}}
   >
-    <div class="modal-outer-container">
-      <div class="modal-middle-container">
-        <div class="modal-inner-container">
-          {{yield to="aboveHeader"}}
-          {{#if
-            (and
-              (not @hideHeader)
```

Comment:
Note to self: looks like we lost this new conditional when resolving merge conflicts 👀

Reply:
Good catch, thanks!